content/v2.0/write-data/_index.md

```diff
@@ -22,12 +23,10 @@ Telegraf, data scrapers, the InfluxDB v2 API, `influx` command line interface (C
 the InfluxDB user interface (UI), client libraries, and third-party technologies.
```

Comment:
Collect and write time series data to InfluxDB Cloud and InfluxDB OSS.

Reply:
Perhaps we could add the following statement to the intro:
Discover how to quickly start collecting data, and then explore other ways to write data using no-code solutions or developer tools.

Then, we could replace the [Other ways to write data] link w [Write data with no-code solutions] & [Write data with developer tools] links

Reply:
It actually has those links in the section under the Other ways to write data heading, so leaving as is.